src/db/migration/migrate20211116-ZmgDevices.pl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/perl
+#
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2015, 2016, 2021 Synacor, Inc.
```

Comment:
we don't need 2015, 2016 as this is new file and added in 2021. If this was existing file and was being updated then earlier years are fine.